packages/ketcher-core/src/application/render/raphaelRender.js

```diff
@@ -226,21 +226,27 @@ Render.prototype.update = function (force = false, viewSz = null) {
       }
     } else {
       const sz1 = bb.sz()
-      const marg = this.options.autoScaleMargin
-      const mv = new Vec2(marg, marg)
+      const { autoScaleMargin, rescaleAmount, maxBondLength } = this.options
+      const mv = new Vec2(autoScaleMargin, autoScaleMargin)
       const csz = viewSz
-      if (marg && (csz.x < 2 * marg + 1 || csz.y < 2 * marg + 1)) {
+      if (
+        autoScaleMargin &&
+        (csz.x < 2 * autoScaleMargin + 1 || csz.y < 2 * autoScaleMargin + 1)
```

Comment:
Maybe it makes sense to put 2 * autoScaleMargin in a variable with a meaningful name?

Reply:
to be honest, I still dont completely understand how exactly this works xP